### PR TITLE
O sa mearga ce scriu aici

### PR DESCRIPTION
**Pull request by Nicolleta1981**

### CODE REVIEW

**Review by biggicode — CHANGES_REQUESTED** [review]
In file names like ''value atribute.html'' you want to put ''-'' between words

File: web/nicolleta1981/html/Value attribute/value attribute.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Value attribute</title>
+  </head>
+
+  <body>
+    <h1>The HTTP method that the browser uses to submit the form can have values bellow:
+      <ul>
+        <li>post-HTTP POST method-form data are included in the body of the form and sent to the server</li>
+        <li>get-HTTP GET method-form data are appended to the action attribute URI with a '?'as separator, and the resulting URI is sent to the server. Use this method when the form has no side-effects and contains only ASCII characters</li>
+        <li>dialog-Use when the form is inside a < dialog > element to close the dialog when submitted.
```
Comment (biggicode) [inline web/nicolleta1981/html/Value attribute/value attribute.html]: you want to close dialog element

File: web/nicolleta1981/html/Value attribute/value attribute.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          <p>This value can be overriden by a formmethod attribute on a < button >or < input >element.</p>
```
Comment (biggicode) [inline web/nicolleta1981/html/Value attribute/value attribute.html]: Button must be closed or deleted here ,on input you want to put atleast a "type" attribute

**Review by vasilemaftita — APPROVED** [review]
Nice; Work.

**Review by vasilemaftita — APPROVED** [review]
Nice; Work!

**Review by screpyy96 — APPROVED** [review]
💯

**Review by vasilemaftita — APPROVED** [review]
Nice; Work!

**Review by psyhedeliq — APPROVED** [review]

**Review by Adrian01C — COMMENTED** [review]
cool

**Review by marianzburlea — APPROVED** [review]

File: web/nicolleta1981/css/font/font.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Let's try chnage as much we can on font</title>
+</head>
+<link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (marianzburlea) [inline web/nicolleta1981/css/font/font.html:9]: baga-le in head

**Review by marianzburlea — APPROVED** [review]

**Review by vasilemaftita — COMMENTED** [review]
Nice; Work...

**Pull request merged**